test/controllers/admin/settings_controller_test.rb

```diff
@@ -0,0 +1,11 @@
+require 'test_helper'
+
+class Admin::SettingsControllerTest < ActionController::TestCase
```

Comment:
Are there tests missing from here? Maybe you can add some for the settings controller in a new PR